components/admin_console/admin_definition.jsx

```diff
@@ -1367,7 +1367,8 @@ const AdminDefinition = {
                         label: t('admin.rate.enableLimiterTitle'),
                         label_default: 'Enable Rate Limiting:',
                         help_text: t('admin.rate.enableLimiterDescription'),
-                        help_text_default: 'When true, APIs are throttled at rates specified below.',
+                        help_text_default: 'When true, APIs are throttled at rates specified below.\n \nRate limiting prevents server overload from too many requests. This is useful to prevent 3rd party applications or malicous attacks from impacting your server.',
```

Comment:
@cwarnermm Can you confirm how we stylize the word "3rd party"?  Do we spell out as in "third party" or is it good as a number?

Reply:
In the Features section for the Link Metadata Timeout, we use the spelling "third-party". Would it be better to use this form?

Reply:
@maxerenberg @wiersgallak  We use "third-party" rather than "3rd party" for clarity.